explainer.md

```diff
@@ -22,118 +71,260 @@ place associated with the application (such as the shelf or home screen).
 * To indicate a small amount of additional information, such as an unread count.
-* To allow the application to convey this information regardless of whether any
-  of the application's windows are open.
+* To allow certain user blessed pages (such as Bookmarks or [Installed Web Applications](https://www.w3.org/TR/appmanifest/#installable-web-applications)) to convey this information, regardless of whether they are currently open.
 
 Non-goals are:
 
 * To provide an arbitrary image badge.
 
 Possible areas for expansion:
 
-* Providing badging for sites in a normal web browsing context. The current
-  proposal is just for installed apps (designed to show up in the operating
-  system shelf area). We could also explore icon badging on the drive-by web.
-  This naturally leads into...
-* Providing per-tab and per-window badging. The current proposal is for a global badge
-  for the application. See [#1](https://github.com/WICG/badging/issues/1).
-* Support apps that want to render a small status indicator (e.g., a music app shows ▶️	
-  or ⏸️; a weather app shows ⛈️ or ⛅️).
-* Setting the badge from a service worker (e.g. an email app updating an unread count).
+* Support rendering a small status indicator (e.g., a music app shows ▶️ or ⏸️; a weather app shows ⛈️ or ⛅️).
+* Support setting the badge from a service worker (e.g. an email app updating an unread count in the background).
+* Support badges on bookmark icons (this is probably up to user agents to support, and may not need anything new API wise).
+* Support for non-inherited badges (see [Issue 42](https://github.com/WICG/badging/issues/42)).
+* Support for unicode glyph badges.
 
 Examples of sites that may use this API:
 
-* Chat, email and social apps, to signal that new messages have arrived.
-* Productivity apps, to signal that a long-running background task (such as
-  rendering an image or video) has completed.
-* Games, to signal that a player action is required (e.g., in Chess, when it is
-  the player's turn).
+* Chat, email and social apps; to signal that new messages have arrived.
+* Productivity apps, to signal that a long-running background task (such as rendering an image or video) has completed.
+* Games, to signal that player action is required (e.g., in Chess, when it is the player's turn).
 
 Advantages of using the badging API over notifications:
 
-* Can be used for much higher frequency events than notifications, because each
-  new event does not disrupt the user.
-* There is no need to request permission to use the badging API, since it is
-  much less invasive than a notification.
+* Can be used for much higher frequency events than notifications, because each new event does not disrupt the user.
+* There is no need to request permission to use the badging API, since it is much less invasive than a notification.
```

Comment:
```suggestion
* There may be no need to request permission to use the badging API, since it is much less invasive than a notification.
```

Reply:
If we are considering a permission, maybe we should have this work via promises after all? @mgiuca 

However, I feel like badging probably doesn't need a permission (sites can already change their favicons without one) and this feels like a pretty minor extension from that point of view.

Reply:
Not right now... just don't want to give the impression that this is all set in stone.

Reply:
All right, I'll the suggestion :)